test/parallel/test-debug-port-cluster.js

```diff
@@ -3,7 +3,7 @@ const common = require('../common');
 const assert = require('assert');
 const spawn = require('child_process').spawn;
 
-const PORT_MIN = common.PORT;
+const PORT_MIN = common.PORT + 1; // fixture uses common.PORT
```

Comment:
I thought so too at first, but it turns out that the fixture uses a single port, while this uses use multiple ports. Less weird IMO to give the `common.PORT` to the fixture and let this use special ports rather than have both using special ports and needing to make sure that the special ports don't collide.